docroot/sites/all/modules/features/registration_pages/registration_pages.features.field_instance.inc

```diff
@@ -683,25 +687,85 @@ function registration_pages_field_default_field_instances() {
     ),
   );
 
+  // Exported field_instance: 'node-registration_page-field_total_tickets'
```

Comment:
If you're using 'field_tickets_for_aa_members' for the ticket count, what is 'field_total_tickets'? I think your instinct is good to replace the field, but you haven't completed the work to implement it. Also, if the form went live before the merge and deployment, what happens to tickets that are sold before the deployment? How will your ticket count work?